state-chain/custom-rpc/src/lib.rs

```diff
@@ -240,9 +247,11 @@ pub enum RpcAccountInfo {
 	Broker {
 		flip_balance: NumberOrHex,
 		bond: NumberOrHex,
+		#[deprecated(note = "This field is deprecated and will be replaced in a future release")]
 		earned_fees: any::AssetMap<NumberOrHex>,
 		affiliates: Vec<(AffiliateShortId, AccountId)>,
```

Comment:
Might as well deprecate this too and add the account_id into RpcAffiliateDetails.